config/crd/bases/llamastack.io_llamastackdistributions.yaml

```diff
@@ -107,8 +107,12 @@ spec:
                                       description: The key to select.
                                       type: string
                                     name:
+                                      default: ""
                                       description: |-
                                         Name of the referent.
+                                        This field is effectively required, but due to backwards compatibility is
+                                        allowed to be empty. Instances of this type with an empty value here are
+                                        almost certainly wrong.
```

Comment:
You mean, the name of the referenced ConfigMap ? Maybe we can add this here instead of just `Name of referent.` (and if so, shouldn't it be `reference` as the referent is the operator itself ?)

Reply:
It looks like that the descriptions here are picked up from somewhere else, also not necessarily coming from the operator code. How have those been added ?